package.json

```diff
@@ -33,6 +33,11 @@
     "voice",
     "voip"
   ],
+  "overrides": {
+    "@types/express-serve-static-core": "4.17.8",
+    "@types/mime": "2.0.2",
+    "@types/express": "4.17.7"
```

Comment:
We don't use express. Why do we need these two express related ones?
Also, why put @types/mime in overrides section? I think this will work if you it under dependencies.

Reply:
I can move `@types/mime` down to dev dependencies, though the express ones are required since without them, we get build errors. In this example, I've simply removed `@types/express` and kept `@types/mime` in dev dependencies, and we get this : 
![image](https://user-images.githubusercontent.com/43423318/215577107-629c61b6-a42b-4365-9f91-e52e54609ac5.png)

So it seems like it's not the SDK that uses express, but `typedoc-plugin-...` that uses it? And if we don't explicitly tell the sdk which version to install (because of the lack of package-lock.json), we simply cannot install the packages for the SDK, worse yet, we were unable to build the SDK with npm.

Reply:
My 2c is to remove the overrides and simply keep a `package-lock.json` to pin the versions that are also nested within dependencies, so we can remove the overrides from the package.json.

Reply:
removing the overrides and keeping package-lock.json sounds a better approach.